lib/computer_player.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "player"
+
+class ComputerPlayer < Player
+
+  attr_reader :player_input
+
+  def get_input
+    @display.present(Messages.new.player_prompt(@name))
```

Comment:
1) Where do we get `@name` from? 
2) You could argue this either way, but I'd probably prefer to see a different class prompt the player to get their input, and to display it once they've chosen. Would be keen to hear other's thoughts on this.

Reply:
@pelensky I could have a new `GetInput` class?

Reply:
Do you have a Display class already @Himalee ?

Reply:
Maybe it could go in Display?